File: Beelina.LIB/GraphQL/Errors/ProductWithdrawalEntryNotExistsError.cs
using Beelina.LIB.Interfaces;

namespace Beelina.LIB.GraphQL.Errors
{
    public class ProductWithdrawalEntryNotExistsError
        : BaseError, IProductWithdrawalEntryPayload
    {
        public ProductWithdrawalEntryNotExistsError(int id)
        {
            Message = $"Product withdrawal entry with the id of {id} does not exists!";
        }
    }
}

File: Beelina.APP/src/app/_models/errors/product-withdrawal-entry-not-exists.error.ts
import { IBaseError } from 'src/app/_interfaces/errors/ibase.error';
import { IProductWithdrawalEntryPayload } from 'src/app/_interfaces/payloads/iproduct-withdrawal-entry-query.payload';

export class ProductWithdrawalEntryNotExistsError
  implements IProductWithdrawalEntryPayload, IBaseError {
  public typename: string;
  public code: string;
  public message: string;
}
